.github/workflows/codeql.yml

```diff
@@ -42,11 +42,42 @@ jobs:
           path: rskj
           fetch-depth: 0
   
+      - name: Determine branch to checkout for PR
+        if: github.event_name == 'pull_request'
+        working-directory: rskj
+        env:
+          PR_BRANCH: ${{ github.head_ref }}
+        run: |
+          IS_RSKJ_BRANCH=$(git ls-remote --heads origin "$PR_BRANCH")
+          if test -n "${IS_RSKJ_BRANCH}"; then
+          echo "Found matching branch name in RSKj repo"
+          CHECKOUT_REF="$PR_BRANCH"
+          else
+          echo "Using master for RSKj"
```

Comment:
Do you mind formatting this to be indented as the one in the section below? Thanks!